demo/src/TestArea.vue

```diff
@@ -46,21 +56,34 @@ export default {
     messageStyling: {
       type: Boolean,
       required: true
+    },
+    chatList: {
+      type: Array,
+      required: true
+    },
+    multipleChatsEnabled: {
+      type: Boolean,
+      required: true
```

Comment:
Why have it as a prop? Is there any difference when this is off? I would expect single instances of the component to work as they have been, and multiple instances to also work out-of-the-box.

Reply:
In multi-room mode, when chat window is opened, I show list of chat rooms (as opposite to chat right away). Also, in the header for chat window I show go-back-to-chat-list button. I use multipleChatsEnabled prop to determine if those 2 things need to be done. I don't see any other good way to do it. The main reason for that is the demo app is controlling the chat content and there is only one property (messageList) with chat content it passes launcher. So, the demo app puts correct chat content when chat room changes. I considered to use chatList prop instead of multipleChatsEnabled (if it's empty or undefined than assume single chat mode), but it's possible to have 0 chat rooms in multi-chat mode.